pylint_odoo/checkers/modules_odoo.py

```diff
@@ -823,17 +823,14 @@ def _get_manifest_referenced_files(self):
                 referenced_files[fname] = data_type
         return referenced_files
 
-    def _get_module_files(self):
+    def _check_file_not_used(self):
+        """Check if a file is not used from manifest"""
+        self.msg_args = []
```

Comment:
Use tuple and use `self. module_files.add()` instead of `self. module_files.append()`